doc/FAQ.md

```diff
@@ -0,0 +1,10 @@
+## Debug
+
+Common problems:
+
+1. When starting RP I get an error:
+```
+InnerError={"code":"ForbiddenByPolicy"} 
```

Comment:
I am getting this :-(

Reply:
this means your Vault policy is wrong. Potentially when you created it the wrong env was configured.